processor/k8sprocessor/testdata/config.yaml

```diff
@@ -58,6 +58,10 @@ processors:
         name: ip
       - from: resource_attribute
         name: k8s.pod.uid
+    
+    ignored_pod_names:
+      - jaeger-agent
+      - jaeger-collector
```

Comment:
Since we already have `filter` config above, I think adding a more generic `exclude` would be better than something dedicated to just ignoring pods. May be something like

```yaml
exclude:
    pods:
        - jaeger-agent
        - jaeger-collector
```

Reply:
Actually I think the following would be even better as it's more future proof. This can allow us to exclude pods by fields other than name in future if we want.

```yaml
exclude:
    pods:
        - name: jaeger-agent
        - name: jaeger-collector
```

Reply:
Sounds good. Will make those changes now :)